templates/partial/_partners-footer.html

```diff
@@ -1,9 +1,32 @@
-<section class="p-strip--white is-deep">
-  <div class="row">
+<section class="p-section">
+  <hr class="p-rule is-fixed-width">
+  <div class="row p-section">
```

Comment:
In the [Vanilla documentation](https://vanillaframework.io/docs/patterns/section) it says that sections should not be nested inside of each other, am I understanding that correctly?

Reply:
Yes, I don't know what is the best course for this. We need this kind of spacing, how we obtain it is a good question. We could either update the vanilla docs, or create a new component or modifier class. cc @bartaz.

The basic idea is this: depending on context, some things might get lost or not have the necessary "gravitas". This is when we use the bigger of the two spacings we have. (The smaller being `p-block`).

Reply:
IMO if there are cases where we're supposed to ignore the documentation, then the documentation should be updated to clearly reflect that

Reply:
If we are putting a section inside section to achieve double bottom margin than it's clearly something wrong.

All it is I believe is a "deep" spacing. So this should not be a double section but a deep strip.
And as far as I remember we discussed that when you need a deep spacing then it's needed both on top and bottom of given part of content (because it needs same "emphasis" from both sides), that's why the deep spacing is only available in deep strips.

So seems like the page content above the footer should be wrapped in a deep strip if we want a deep spacing between content and footer. But I guess we will again end up in a point there this strip will only need deep spacing on bottom and not on top (which is what we wanted to avoid with new components)… So we seem to be running in circles a bit again.

Allowing nesting sections in each other is a red flag for me, and it should be the last resort. In this case I believe this can be solved by using deep spacing (so a deep strip instead). I guess the remaining question for @lyubomir-popov is, is this still true that with deep spacing we always need it both on top or bottom, or are there cases (like this one likely?) when we only need deep spacing on the bottom. If so, we need to add deep section I guess?

Reply:
If the reason for this is to create a visible disconnect between the content and the footer. Could we simply add margin-top to footers across all sites?